data/adapters/VAMP_coding_variant_scores_adapter.py

```diff
@@ -81,9 +85,11 @@ def process_coding_variant_phenotype_chunk(self, chunk, type='hgvsp'):
                         'name': self.PHENOTYPE_EDGE_NAME,
                         'inverse_name': self.PHENOTYPE_EDGE_INVERSE_NAME,
                         'files_filesets': 'files_filesets/' + self.file_accession,
-                        'simple_sample_summaries': self.igvf_metadata_props.get('simple_sample_summaries'),
-                        'method': self.igvf_metadata_props.get('method'),
-                        'biological_context': self.igvf_metadata_props['samples'][0] if 'samples' in self.igvf_metadata_props else None,
+                        'simple_sample_summaries': file_fileset_obj.get('simple_sample_summaries'),
```

Comment:
we decide to use value of file_fileset_obj['samples'][0] for field "biosample_term", use value of file_fileset_obj.get('simple_sample_summaries')[0] for field "biological_context". remove field "simple_sample_summaries"